lib/screens/explore_screens/explore/explore_screen.dart

```diff
@@ -78,10 +86,19 @@ class ExploreScreen extends StatelessWidget {
                     Expanded(
                       child: ExploreCard(
                         onTap: () {
-                          BlocProvider.of<ExploreBloc>(context).add(const OnExploreCardTapped(Routes.vouch));
+                          showDialog<void>(
+                            context: context,
+                            builder: (_) {
+                              return const FlagUserInfoDialog();
+                            },
+                          ).whenComplete(
+                            () => ScaffoldMessenger.of(context).showSnackBar(
+                              SnackBar(content: Text("Coming up! ".i18n), duration: const Duration(seconds: 1)),
+                            ),
+                          );
```

Comment:
I made a PR that change Explore cards maybe this dialog could be launched by a listener

Reply:
I like what you did there, ok ill merge mine after and use your changes